site/content/integrate/apps/lifecycle/callbacks.md

```diff
@@ -3,5 +3,15 @@ title: Callbacks
 description: Methods that the Apps plugin will invoke on the App
 weight: 10
 ---
-### Coming soon!
-In the future this page will describe the various callback functions that the Apps plugin may call at some point in the App's lifecycle. For example: `OnInstall`, `OnUninstall`.
+The App [manifest]({{<ref "/integrate/apps/structure/manifest">}}) defines a number of callback methods ([Calls]({{<ref "/integrate/apps/structure/call">}})) that are invoked by the Mattermost server at various times. If a callback isn't defined in the manifest, it won't be executed by the Mattermost server.
+The following table lists supported callbacks:
+
+| Name                | Manifest Parameter       | Details                                                                                                                                                                                                                                                                                                                                                                             |
+|---------------------|--------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OnInstall           | `on_install`             | OnInstall gets invoked when a System Admin installs the App with a `/apps install` command. It may return another call to the app, or a form to display.                                                                                                                                                                                                                            |
```

Comment:
On the other page, all of these names were inside `code blocks`, so because they're not here, they look out of place. I think it should go in a `code block`, both here and in the details.

Reply:
Agreed. Will do :)